server/src/main/scala/services/graphstages/WorkloadCalculator.scala

```diff
@@ -98,7 +110,7 @@ object WorkloadCalculator {
 
     val splitWorkLoadInMinute = (apiSplitRatio.nationalities, useNationalityBasedProcTimes) match {
       case (Some(nats), true) if nats.values.sum > 0 =>
-        val bestPax = PcpPax.bestPax(arrival)
+        val bestPax = PcpPax.bestPaxEstimateWithApi(arrival)
```

Comment:
should this `Pcp.bestPaxEstimateWithApi` be replaced by the `pcpPaxFn` passed in?

Reply:
nice catch, I'd better add a test for this.